### ISSUE TITLE
Pseudo-classes have wrong names

### ISSUE THREAD

**Issue by MartinKavik** [justgage/tachyons-elm#1]
Hi,

#### Bug example
`.pointer:hover` (see [tachyons styles](http://tachyons.io/docs/table-of-styles/)) has been transformed to:
```
{-| pointerd
-}
pointerd : String
pointerd =
    "pointerd"
``` 

---

#### Questions / ideas to discuss in other issues

- Could you, please, explain how you generated **Tachyons.elm**?
  - I develop in *Visual Studio Code* and I want to generate some docs informations (I want to show CSS properties of the chosen Tachyons class) in *Code*'s autocomplete window
- Could you separate Tachyons classes and helper functions to the corresponding modules (for example `Tachyons.Classes` and `Tachyons.Tachyons`)? Why:
  - Function `classes` has very general name - I often want to use it only as `Tachyons.classes`, because it can be confusing if other libraries have function with the same name or I want to use variable `classes`
  -  I think It will be a little bit easier to regenerate Tachyons classes from new Tachyons versions
- Do you use some patterns / best practices when you develop with this library?
  - I've written a basic prototype of a button component: [github.com/MartinKavik/tachyons-elm-ui-example](https://github.com/MartinKavik/tachyons-elm-ui-example)

Thank you very much!

**Comment by justgage** [comment]
Wow, I think this is my first issue on one of my open source projects 😁. 

I was afraid that everything don't get translated properly. Sorry about that. Really I just did it with some macros one night. What I would love to do it's generate this using a CSS parser some day. Not sure if I'll have time in the near future but if you want to make a PR I would be grateful. 

Yeah breaking it up into different modules is a good idea. Not sure a better name than classes... Maybe just alias Tachyons to T and then it's shorter? 

As for best practices I would mostly look at the main website. Maybe join they're slack. I personally just try to group similar things like spacing, color, etc...

**Comment by justgage** [comment]
I've published a new version using @nakamin's PR #2 in #3. You can view the new version here: http://package.elm-lang.org/packages/justgage/tachyons-elm/2.0.0/Tachyons-Classes. I think this should fix the issues you've experienced, however if there are other problems please comment on this issue.
